keras/src/layers/rnn/gru_test.py

```diff
@@ -400,3 +400,33 @@ def test_legacy_implementation_argument(self):
             tpu_atol=1e-3,
             tpu_rtol=1e-3,
         )
+
+    def test_stateful_with_symbolic_batch_size(
+        self,
+    ):
+        from keras.src import backend
+
+        if backend.backend() != "tensorflow":
+            self.skipTest(
+                "Test only applicable to fixing a bug for TensorFlow backend."
+            )
```

Comment:
Use `@pytest.mark.skipif(` to decorate the test instead of skipping within the test